app/main/routes.py

```diff
@@ -51,7 +50,6 @@
     generate_pdf_manifest,
     get_download_filename,
     get_file_details,
-    get_file_mimetype,
```

Comment:
As mentioned in the other PR, since you are removing the only use of this function `get_file_mimetype` here, you can go ahead and delete the function definition itself as well